examples/07_end_to_end/seismic_waveform.ipynb

```diff
@@ -0,0 +1,1322 @@
+       "           calib: 1.0\n",
+       "    back_azimuth: 100.0\n",
+       "     inclination: 30.0\n",
+       "        response: Channel Response\n",
+       "\tFrom M/S (Velocity in Meters Per Second) to COUNTS (Digital Counts)\n",
+       "\tOverall Sensitivity: 2.5168e+09 defined at 0.020 Hz\n",
+       "\t4 stages:\n",
+       "\t\tStage 1: PolesZerosResponseStage from M/S to V, gain: 1500\n",
+       "\t\tStage 2: CoefficientsTypeResponseStage from V to COUNTS, gain: 1.67785e+06\n",
+       "\t\tStage 3: FIRResponseStage from COUNTS to COUNTS, gain: 1\n",
+       "\t\tStage 4: FIRResponseStage from COUNTS to COUNTS, gain: 1"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "tr.stats"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "0f8fde18-2112-4e60-bf56-a377ae1f0db3",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([0.        , 0.00694644, 0.07597424, ..., 1.93449584, 0.98196204,\n",
+       "       0.44196924])"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "tr.data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1faba8a2-25ce-4610-b8ab-9635d7b9ea80",
+   "metadata": {},
+   "source": [
+    "## Simulation\n",
+    "In real-time, trace data will be sent in batches of observations, so we can use csp to simulate this by writing a node that divides a stream into a time series of pieces based on an interval, and assuming a certain transmission lag"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "03cea2e1-c58f-4835-8b91-3aa043b857b7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@csp.graph\n",
+    "def trace_sim(stream: Stream, interval: timedelta, lag: timedelta = timedelta(0)) -> Dict[str, ts[Trace]]:\n",
+    "    out = {}\n",
+    "    for trace in stream:\n",
+    "        # Approximate number of equal intervals. This could be improved by using .slice over time intervals\n",
+    "        num_intervals = int((trace.stats.endtime - trace.stats.starttime) / interval.total_seconds())\n",
+    "        # Create a new stream of the split trace\n",
+    "        if num_intervals > 1:\n",
+    "            trace_stream = trace / num_intervals\n",
+    "        else:\n",
+    "            trace_stream = Stream(trace)\n",
+    "        # Create a curve of the trace that ticks some lag after the end time\n",
+    "        out[trace.id] = csp.curve(Trace, [(t.stats.endtime.datetime + lag, t) for t in trace_stream])\n",
+    "    return out"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "199cc6f0-7103-4d93-8e4b-ffbf17d07200",
+   "metadata": {},
+   "source": [
+    "We can now run this simple graph to show how the pieces are divided into 5 second chunks with a small simulated transmission delay"
```

Comment:
I would make a note in this text block why one sees 3 data chunks per interval (it's the 3 coordinate axes, N, E and Z).
```
2009-08-24 00:20:09 Received simulated trace BW.RJOB..EHZ:BW.RJOB..EHZ | 2009-08-24T00:20:03.000000Z - 2009-08-24T00:20:08.990000Z | 100.0 Hz, 600 samples
2009-08-24 00:20:09 Received simulated trace BW.RJOB..EHN:BW.RJOB..EHN | 2009-08-24T00:20:03.000000Z - 2009-08-24T00:20:08.990000Z | 100.0 Hz, 600 samples
2009-08-24 00:20:09 Received simulated trace BW.RJOB..EHE:BW.RJOB..EHE | 2009-08-24T00:20:03.000000Z - 2009-08-24T00:20:08.990000Z | 100.0 Hz, 600 samples
```